evm_loader/lib/src/account_storage.rs

```diff
@@ -342,6 +439,18 @@ impl<'a, T: Rpc> EmulatorAccountStorage<'_, T> {
         Ok(())
     }
 
+    fn is_storage_overriden(&self, pubkey: Pubkey) -> bool {
```

Comment:
Overrides is an Ethereum construct, they are associated with an Ethereum account address.
Pubkey is a Solana account identifier. Please avoid mixing Solana and Ethereum types whenever possible.
You have and Ethereum account address where this function is called.

Reply:
done